cli/pkg/release/gbm.go

```diff
@@ -49,11 +52,11 @@ func CreateGbmPR(version, dir string) (gh.PullRequest, error) {
 			return pr, err
 		}
 
-		console.Info("Set upstream to trunk", org)
-		err = git.SetUpstreamTo("trunk")
-		if err != nil {
-			return pr, err
-		}
+		// console.Info("Set upstream to trunk", org)
+		// err = git.SetUpstreamTo("trunk")
+		// if err != nil {
+		// 	return pr, err
+		// }
```

Comment:
I don't believe that it's actually necessary to set the upstream to `trunk` here -- likely it could be removed.

Reply:
Good call. I wonder if we also need the `AddRemote` call above ?
I only remember running into git setup issues with the Gutenberg submodule. I would think since we are cloning from the remote repo, git would set that up for us by default.